### DISCUSSION TITLE
Hyperlink to download Portable version broken (v2.17.3)

### DISCUSSION THREAD

**Discussion by pranamg** [DaxStudio/DaxStudio#768] [General]
I am unable to locate the Portable Version of DAX Studio (v2.17.3) on the Github Repository. It seems that the "DaxStudio_2_17_3_portable.zip" file has been moved. The hyperlink on "**[https://daxstudio.org/downloads/](https://daxstudio.org/downloads/)**" directing me to the Portable Version download link throws up the error "**Error 503 No healthy IP available for the backend**". The folder structure listed in the hyperlink, no longer seems to exist. (["https://github.com/DaxStudio/DaxStudio/releases/download/v2.17.3/DaxStudio_2_17_3_portable.zip"]("https://github.com/DaxStudio/DaxStudio/releases/download/v2.17.3/DaxStudio_2_17_3_portable.zip"))

**Comment by dgosbell** [comment]
I think this must have been a temporary issue on github. I just checked and the download link is working fine for me.
